src/log_batch.rs

```diff
@@ -876,8 +921,7 @@ impl LogBatch {
     }
 }
 
-/// Verifies the checksum of a slice of bytes that sequentially holds data and
-/// checksum.
+/// Verifies the checksum of a slice of bytes that sequentially holds data.
```

Comment:
```suggestion
/// Verifies the checksum of a slice of bytes that sequentially holds data and checksum.
```

Reply:
No need for it.
* `Verify_checksum` is the original func just for checking whether the content of entries in the LogBatch is valid or not. 
* And `Verify_checksum_with_context` is the verifier with checksum when decoding the whole `LogBatch`.
And the annoatations to them are specified these difference respectively.

Reply:
What do you mean no need? It's wrong.

The comment explains two things: (1) the content of input buffer, which is [data][checksum] (2) the functionality, which is verifying the checksum.

You version is wrong in explaining the (1).